.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+## What
+
+If the PR tries to resolve any Open Issue, make sure to include the text.
+**The PR tries to resolve #<ISSUE-NUMBER>.**
+
```

Comment:
Nice work with combining PR of both repos. Let's add para (of description) here by moving current para above **What** section.

Reply:
Can you please elaborate? I am confused.